assignments/prototypes.js

```diff
@@ -15,6 +15,14 @@
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
 
+function GameObject(gameObjectAttributes) {
+  this.createdAt = gameObjectAttributes.createdAt;
+  this.dimensions = gameObjectAttributes.dimensions;
+}
+  GameObject.prototype.destroy = function() {
+    return 'Object was removed from the game.';
+  }
```

Comment:
So this is supposed to print the actual name of the object, not the word "Object". Try something like `this.name` instead.